[2025-02-17T18:38:23.557+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: etl_pipeline.extract manual__2025-02-17T18:38:22.524720+00:00 [queued]>
[2025-02-17T18:38:23.572+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: etl_pipeline.extract manual__2025-02-17T18:38:22.524720+00:00 [queued]>
[2025-02-17T18:38:23.574+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-02-17T18:38:23.575+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-02-17T18:38:23.576+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-02-17T18:38:23.600+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): extract> on 2025-02-17 18:38:22.524720+00:00
[2025-02-17T18:38:23.605+0000] {standard_task_runner.py:55} INFO - Started process 372 to run task
[2025-02-17T18:38:23.610+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'extract', 'manual__2025-02-17T18:38:22.524720+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpuou297ty']
[2025-02-17T18:38:23.613+0000] {standard_task_runner.py:83} INFO - Job 2: Subtask extract
[2025-02-17T18:38:23.633+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-17T18:38:23.708+0000] {task_command.py:388} INFO - Running <TaskInstance: etl_pipeline.extract manual__2025-02-17T18:38:22.524720+00:00 [running]> on host ffeed52c0e5e
[2025-02-17T18:38:23.811+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_pipeline
AIRFLOW_CTX_TASK_ID=extract
AIRFLOW_CTX_EXECUTION_DATE=2025-02-17T18:38:22.524720+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-17T18:38:22.524720+00:00
[2025-02-17T18:38:23.813+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2025-02-17T18:38:23.814+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "Extrayendo datos..."']
[2025-02-17T18:38:23.824+0000] {subprocess.py:86} INFO - Output:
[2025-02-17T18:38:23.825+0000] {subprocess.py:93} INFO - Extrayendo datos...
[2025-02-17T18:38:23.826+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-17T18:38:23.863+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=extract, execution_date=20250217T183822, start_date=20250217T183823, end_date=20250217T183823
[2025-02-17T18:38:23.903+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-02-17T18:38:23.961+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
